src/components/Resources/index.js

```diff
@@ -56,20 +33,20 @@ function Resources({ getResourcesUrl }) {
         {searchValue && (
           <Typography>
             You have searched for "<strong>{searchValue}</strong>" and gotten
-            <strong> {count}</strong> results.
+            <strong> {data.count}</strong> results.
           </Typography>
         )}
         <br />
-        {loading && !errorMessage ? (
+        {isLoading && !isError ? (
```

Comment:
Let's handle the error and loading state like how it's handled above: 
```js
 if (isLoading) {
    return <p>Loading...</p>;
}
```

Reply:
Whoops, I just realized that the `isLoading && !isError` is leftover logic, but since we already have an `isLoading` on the page we don't need two. So this is the change I made: b5e0335 Is that acceptable?